configs/envoy_service_to_service.template.json

```diff
@@ -156,6 +156,47 @@
         ]
       }
     }]
+  },
+  {
```

Comment:
Added a small comment in the template and added a note to the reference configuration docs.